TaxFoundation/tobacco-taxes
# update data and readme for better clarity on update process

#Details
This PR updates the data with the most recent 2023 numbers, and also overhauls the read me file to make updating data easier going forwards.

#Steps to Test
1. open repo in VS Code. 
2. Download the "Live Server" extension
3. open the index.html file and click the Go Live button on the blue bar at the bottom of VS Code.
4. Ensure line extends past 2020, the last year in the previous version and that the inflation checkbox works without causing the graph to go haywire.
5. Look at new readme file and ensure instructions for further updates are legible and correct.

#Linked tasks
[#863gtw4g4](https://app.clickup.com/t/863gtw4g4)
